src/renderer/store/playback/actions.ts

```diff
@@ -0,0 +1,24 @@
+import { Action } from '../action';
+
+export const VIDEO_PLAYING = 'VIDEO_PLAYING';
+export const VIDEO_SEEK = 'VIDEO_SEEK';
+export const VIDEO_SKIP = 'VIDEO_SKIP';
+
+export const videoPlaying: (playbackPlaying: boolean) => Action<boolean> = (
+  playbackPlaying
+) => ({
+  type: VIDEO_PLAYING,
+  payload: playbackPlaying,
+});
+
+export const videoSeek: (playbackTime: number) => Action<number> = (
```

Comment:
regarding comment in the next file - `new Date()` would be called in this function, for example (as well as in `videoSkip` and `videoPlaying`)

Reply:
Hi Chloe, I made the changes requested. Let me know if you see anything else which needs to be changed or if I implemented anything incorrectly. Thanks.